src/components/Pagination/StyledPagination.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components'
+import Button from '../Button'
+import { Paragraph } from '../Paragraph'
+import {
+  genericStyles,
+  breakpointStyle,
+  responsiveProps,
+} from '../../utils/helpers'
+import { breakpoints } from '../../utils/constants'
+
+const StyledPagination = styled.div`
```

Comment:
I think it would be semantically better (and closer to RAD) to mark this up as a `nav` element

Reply:
I think this should be a `Col` component `as` a `nav` then